proto/astarteplatform/msghub/message_hub_error.proto

```diff
@@ -0,0 +1,29 @@
+/*
+* This file is part of Astarte.
+*
+* Copyright 2024 SECO Mind Srl
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*   http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*
+* SPDX-License-Identifier: Apache-2.0
+*/
+
+syntax = "proto3";
+
+package astarteplatform.msghub;
+
+/* MessageHub error type */
+message MessageHubError{
+  string description = 1;     // Human-readable string describing the error.
+  repeated string source = 2; // A list specifying general categories of Astarte Message Hub error.
```

Comment:
I would add here that this will be used as a backtrace for the error.

Reply:
Changed 👍🏻